src/bot/scripts/card.js

```diff
@@ -728,946 +66,1233 @@ const card = new Command({
+                // -aoe attack type
+                // -aoe damage percentage
+                // -aoe radius
+                // -aoe knockback percentage
+
+                // ***POWER INFO***
+
+                // -targeting ...
+                // -power duration
+
+                // ***SPEED INFO***
+
+                // -time to max velocity
+                // -max velocity
+
+                // ***totem ONLY***
+
+                // -health loss
+
+                // ***spell ONLY***
+
+                // *** ??? ***
+
+                // -requirements ...
+                // -child unit limit
+
+                // *name
+                // CanAttack
+                // *description
+                // *Image
+                // -ManaCost
+                // -damage
+                // -health
+                // healthLoss - character_type: totem
+                // type - if !Character, character_type === undefined
+                // Targeting ... - power radius
+                // character_type
+                // AttackRange
+                // TimeToMaxVelocity
+                // MaxVelocity
+                // TimeInBetweenAttacks
+                // PowerDuration
+                // -PowerPower
+                // -rarity
+                // -theme
+                // Requirements
+                // AOEAttacktype
+                // AOEdamagePercentage
+                // AOERadius
+                // AOEKnockbackPercentage
+                // PreAttackDelay
+                // CastArea - ownside/anywhere
+                // ChildUnitLimit
+
+                await message.channel.send("", {
+                  file: path.join(__dirname, "temp", `pd-${saveDate}.png`)
+                });
+                message.channel.send(embed);
+                fs.unlink(
+                  path.join(__dirname, "temp", `pd-${saveDate}.png`),
+                  error => {
+                    if (error !== null && error !== undefined) {
+                      throw `could not delete: pd-${saveDate}.png`;
+                    }
+                  }
+                );
+              }
+            );
+          })
+          .catch(error => {
+            console.error(error);
+            const discordEmbed = new discord.RichEmbed()
+              .setColor("#F8534F")
+              .setTitle(":x: Error 1. Please contact the developer");
+            message.channel.send(discordEmbed);
+          });
+      })
+      .catch(error => {
+        console.error(error);
+        const discordEmbed = new discord.RichEmbed()
+          .setColor("#F8534F")
+          .setTitle(":x: Error 2. Please contact the developer");
+        message.channel.send(discordEmbed);
       });
-    });
   },
 
   load: function () {
     return new Promise((resolve, reject) => {
-
       const promises = [];
 
-      promises.push(jimp.loadFont(path.join(__dirname, "..", "assets", "cards", "fonts", "SP-16.fnt"))
+      promises.push(
```

Comment:
your formatting has fucked up the indentation in this section, pls fix

Reply:
isnt that how they should look when i formatted? How should they look?

Reply:
Uhh how should they look?

Reply:
idk, just make sure that eslint doesnt complain
![image](https://user-images.githubusercontent.com/18635887/71322010-fcf9cb00-24b9-11ea-873c-904a3382f072.png)

Reply:
done